apps/artist/routes.coffee

```diff
@@ -5,7 +5,7 @@ fs = require 'fs'
 request = require 'superagent'
 Backbone = require 'backbone'
 ReferrerParser = require 'referer-parser'
-{ APPLICATION_NAME, NODE_ENV, SHOW_ARTIST_CTA_CODE } = require '../../config'
```

Comment:
Here we no longer need to bootstrap the weird query param we were using to test. The split test component will always just stick what we need into sharify.